app/service/app_svc.py

```diff
@@ -164,7 +178,7 @@ async def validate_requirement(self, requirement, params):
                 self.log.warning(msg)
             else:
                 self.log.error(msg)
-            self._errors.append(Error('requirement', '%s version needs to be >= %s' % (requirement, params['version']), params.get('optional')))
+            self._errors.append(Error('requirement', '%s version needs to be >= %s' % (requirement, params['version'])))
```

Comment:
Same question here about the removal of the operations key.

Reply:
Same answer as above, I *think* you changed it?